tests/components/sensor/test_filesize.py

```diff
@@ -0,0 +1,58 @@
+"""The tests for the filesize sensor."""
+import unittest
+import os
+
+from homeassistant.components.sensor.filesize import CONF_FILE_PATHS
+from homeassistant.setup import setup_component
+from tests.common import get_test_home_assistant
+
+
+TEST_DIR = os.path.join(os.path.dirname(__file__))
+TEST_FILE = os.path.join(TEST_DIR, 'mock_file_test_filesize.txt')
+
+
+def create_file(path):
```

Comment:
Use the `tempfile` module that is part of Python.

Reply:
Actually, it's fine.

Reply:
ok